package.json

```diff
@@ -39,6 +40,7 @@
     "@types/react-transition-group": "^4.4.0",
     "@typescript-eslint/eslint-plugin": "^2.26.0",
     "@typescript-eslint/parser": "^2.26.0",
+    "antd": "^4.1.1",
```

Comment:
This is a really heavy dependency to be adding here - it adds more than 50% to the bundle.

I'll see whether I can get rid of it.

Reply:
In the context of this package I totally agree that using `rc-tooltip` is better than adding the full `antd` package, however viewing it from overall PostHog perspective, we're actually introducing an additional dependency (that also doesn't have the exact same visuals that we already have in the rest of the app). What I think would be the best approach here is to use AntD's tooltip (if the package is available in the target project), and if not use the default browser's tooltip mechanism. Thoughts?

Reply:
Relevant for discussion, https://github.com/PostHog/posthog/issues/2556